### ISSUE TITLE
two-tailed p value

### ISSUE THREAD

**Issue by gobbios** [AlexMielke1988/NetFACS#16]
The way the 'two-tailed' p values are currently defined still bothers me, a lot. In essence, it's just calculated as the smaller of the two one-tailed p values, which effectively makes it a one-tailed p value (except, it's not possible to tell whether it's the right or the left tail without looking at z). I think the much cleaner and transparent solution is to have both, left and right tail values, reported in the function output. 

Plus, I can't imagine a situation where someone would be interested in a two-tailed hypothesis in the first place (even if there was a proper way to actually calculate two-tailed _p_s in this context). My guess is that the vast majority of cases would cover questions like does X occur more often than expected under the alternative model (right tail). Maybe sometimes it could be informative to look into the 'opposite': does X occur less often than expected (left tail). But both at the same time? I find that hard to imagine. Also, in the other functions that rely on this info (those that have a `significance` argument, e.g. `netfacs_network`) as far as I can see the data are filtered in such a way that only 'significant' values with *positive* _z_ are considered, which makes it one-tailed. So even if we return 'two-tailed' (as currently defined) the follow-up functions consider only right-tailed, which makes the whole thing even more confusing (to me at least).

Long story short. I think the best solution is to report two _p_-values, right and left, and remove the `tail` argument again.

**Comment by AlexMielke1988** [comment]
Sorry, but I disagree with you. We are testing many elements using the same dataset at the same time, using bootstraps, and different elements in the same analysis will go in different directions. We are interested whether the observed probability lies outside of the range of the expected values for this sample, or inside. We cannot presuppose whether an element is more or less common than the expected values, and we cannot presupposed what question the users want to ask. All the things we do later with the networks assume a certain interpretation (which elements are more common), but that is not the only question that can be asked with the bootstrapping approach. For example, I am using the same approach currently to test which body parts chimpanzees groom in different conditions, and both higher and lower than expected values are of importance. 

Having one p-value and then an estimate of the effect size (the z value) to interpret the direction of the effect is much more in line with how these values are reported in statistical tests.  Putting two p-values, where essentially p2 = 1 - p1, will confuse the hell out of people and look messy, and I have never seen this in any publication or in any interpretation of a bootstrap or permutation analysis. As it is now, the user decides which interpretation they want. If you want to use one-tailed tests when you apply the package, that is entirely up to you.
